src/main/resources/edu/cornell/kfs/vnd/document/workflow/CuVendorMaintenanceDocument.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<data xmlns="ns:workflow" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="ns:workflow resource:WorkflowData">
+  <documentTypes xmlns="ns:workflow/DocumentType" xsi:schemaLocation="ns:workflow/DocumentType resource:DocumentType">
+    <documentType>
+      <name>PVEN</name>
+      <parent>VEND</parent>
+      <label>Vendor</label>
+      <authorizer>org.kuali.kfs.krad.workflow.authorizer.CfDocumentTypeAuthorizer</authorizer>
+      <helpDefinitionURL>${kfs.externalizable.help.url}/default.htm?turl=WordDocuments%2Fvendorpven.htm</helpDefinitionURL>
+      <active>true</active>
+      <routingVersion>2</routingVersion>
+      <routePaths>
+        <routePath>
+          <start name="AdHoc" nextNode="Submitter" />
+          <role name="Submitter" nextNode="RequiresApproval"/>
+          <split name="RequiresApproval">
+            <branch name="True">
+              <role name="Management" nextNode="Initiator" />
+              <role name="Initiator" nextNode="Join" />
+            </branch>
+            <branch name="False">
+              <simple name="Do Nothing" nextNode="Join" />
+            </branch>
+            <join name="Join" />
+          </split>
+        </routePath>
+      </routePaths>
+      <routeNodes>
+        <start name="AdHoc" />
+        <split name="RequiresApproval">
+          <type>org.kuali.kfs.sys.document.workflow.SimpleBooleanSplitNode</type>
+        </split>
+        <role name="Submitter">
+          <!--<ruleTemplate>KualiVendorDocumentTypeTemplate</ruleTemplate>-->
```

Comment:
Can these commented-out elements be removed?

Reply:
These were in the exported xml but I can remove them.